probability-calculator/prob_calculator.py

```diff
@@ -0,0 +1,54 @@
+import random
+import copy
+
+class Hat:
+    def __init__(self, **all_items):
+        self.contents = []
+
+        for key, value in all_items.items():
+
+            for n in range(value):
+                self.contents.append(key)
+
+        print(self.contents)
+        self.intial_contents = copy.copy(self.contents)
```

Comment:
1. Typo error
2. What's the difference between `contents` and `initial_contents` .... should we make `initial_contents` a const